Makefile

```diff
@@ -23,14 +31,14 @@ endif
 
 .build/deps-build-go.ok:
 	@mkdir -p .build
-	go get -u github.com/golang/dep/cmd/dep
-	go get -u github.com/jteeuwen/go-bindata/...
-	go get -u github.com/elazarl/go-bindata-assetfs/...
+	$(GO) get -u github.com/golang/dep/cmd/dep
```

Comment:
There no need to use `$(GO)` here because there no need for any extra envs passed to the go binary

Reply:
But those are practices that Prometheus developers setup for the projects they maintain, there is no need to make intrusive workflow changes here just to comply with other projects practices